src/Dialect/ONNX/ONNX.td

```diff
@@ -192,18 +184,28 @@ def ONNXBatchNormalizationInferenceModeOp: ONNX_Op<"BatchNormalizationInferenceM
            DefaultValuedAttr<F32Attr, "0.9">:$momentum);
   let results = (outs AnyTypeOf<[AnyMemRef, AnyTensor]>:$o_Y);
   let extraClassDeclaration = [{
-    static int getNumberOfOperands() {
-      return 5;
-    }
-    static int getNumberOfResults() {
-      return 1;
-    }
-    static std::vector<int> getTypeMap() {
-      return {20};
-    }
+    static int getNumberOfOperands() { return 5; }
+    static int getNumberOfResults() { return 1; }
+    static std::vector<int> getTypeMap() { return {20}; }
   }];
 }
 
+def ONNXNoneOp : ONNX_Op<"NoValue", [ConstantLike, NoSideEffect]> {
+  let summary = "An operation representing the absence of a value.";
+  let description = [{
+    This operation can be used to represent the absence of a value. It is typically 
+    used as an argument to operators that have optional parameters.
```

Comment:
Would be helpful if we add an example here, say
```mlir
%cst = "onnx.NoValue"() {value} : () -> none
```

so that uses know `onnx.NoValue` is created by ONNXNoneOp.